pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.black]
+# we use all the defaults
+
+[tool.pytest.ini_options]
+testpaths = [
+  "src/tests/unit_tests",
+]
+addopts = [
+  "--cov",
+]
+
+[tool.coverage.run]
+omit = [
+  "src/tests/*"
+]
+
+[tool.mypy]
+python_version = "3.8"
+
+[[tool.mypy.overrides]]
+module = [
+  "apis.*",
+  "config.*",
+  "elasticsearch",
+  "flask_restx",
+  "flask",
+  "flask_cors",
+  "git",
+  "lxml.*",
+  "lxml",
+  "importer.*",
+  "models.*",
+  "mockito",
+  "numpy",
+  "urllib3",
+  "ontospy.*",
```

Comment:
>ontospy.*",
I recently removed all the ontospy code from the repo. So this module can be removed here too. Ontospy is depending on some older packages that are causing trouble. already, We created a separate repo 'beng-lod-ontospy' to try out some solutions for the ontology visualisation.